cluster/addons/cluster-monitoring/googleinfluxdb/heapster-controller-combined.yaml

```diff
@@ -1,12 +1,16 @@
 {% set base_metrics_memory = "140Mi" -%}
 {% set metrics_memory = base_metrics_memory -%}
+{% set base_metrics_cpu = "80m" -%}
+{% set metrics_cpu = base_metrics_cpu -%}
 {% set base_eventer_memory = "190Mi" -%}
 {% set eventer_memory = base_eventer_memory -%}
 {% set metrics_memory_per_node = 4 -%}
+{% set metrics_cpu_per_node = 0.5 -%}
 {% set eventer_memory_per_node = 500 -%}
 {% set num_nodes = pillar.get('num_nodes', -1) -%}
 {% if num_nodes >= 0 -%}
   {% set metrics_memory = (200 + num_nodes * metrics_memory_per_node)|string + "Mi" -%}
+  {% set metrics_cpu = (80 + num_nodes * metrics_cpu_per_node)|string + "m" -%}
```

Comment:
Is this equation sufficient? Looking at some sample num_node values:

1 node -> 80 + 1*0.5 = 80.5m
10 nodes -> 80 + 10*0.5 = 85m
100 nodes -> 80 + 100*0.5 = 130m
1000 nodes -> 80 + 1000 * 0.5 => 580m

The delta between 10 and 100 nodes is roughly 45m (is that enough?), while from 100 to 1000 is only 450m, which doesn't seem enough. I wonder if we need to have adjustments for various "tiers" of nodes.